nrf52-hal-common/src/spim.rs

```diff
@@ -322,6 +397,7 @@ pub struct Pins {
 pub enum Error {
     TxBufferTooLong,
     RxBufferTooLong,
+    DMABufferInUnaccessableMemory,
```

Comment:
Typo: unaccess**i**ble

Although Google suggests that **in**accessible is the more common word, by far.